transcom/mymove
# INT-B-20286: Partial PPM labeling change in Weight review

## [B-20286](https://www13.v1host.com/USTRANSCOM38/story.mvc/summary?oidToken=Story%3A972106)

## Summary

**INTEGRATION**

This is a simple change of PPM departure date label in Review shipment weight section. It was observed that nothing was needed for HHG component of the AC of the backlog ticket; therefore, nothing was done for this PR where the AC states a "-" must be displayed if GHC has not set the actual delivery date.

[Related PR](https://github.com/transcom/mymove/pull/13295)

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test
The testing involves partial PPM shipment type. This occurs when HHG is involved with a PPM for a given move. 

1. Login as customer and create and submit PPM and HHG.
2. Login as as service counselor and fill in all required info for Order and PPM to click "Submit Move Details".  
3. Login as customer and Upload PPM documentations. Submit.
4. Login as service counseling. Click "Review shipment weights". You shall see "Weight moved by customer". Verify "Actual Departure date" label is present with the date used in the PPM upload documentations process by customer. Also note, HHG   section under "Weight moved". For HHG, the "Delivery date" has not been set yet. Until it is set, a dash "-" will be displayed. 
![Screenshot 2024-07-24 at 2 20 49 PM](https://github.com/user-attachments/assets/40989623-7a36-4625-996c-a574457466d8)
5. Login as TOO. Click checkbox for HHG and click "Approve selected". This will make the move be visible for the GHC/Prime Simulator UI side.
![Screenshot 2024-07-24 at 2 24 00 PM](https://github.com/user-attachments/assets/1729edb1-b51b-409a-948d-f04bac07d73f)
6. Goto Prime Simulator UI.  Click "Update Shipment" for HHG. Fill in all necessary dates ie.. delivery date(make note of date used) ...and save.
7. Goto service counselor and click "Review shipment weights" again. Under HHG, this time you will see a date displayed for delivery date.
![Screenshot 2024-07-24 at 2 29 23 PM](https://github.com/user-attachments/assets/ca577fa5-d2df-4e97-a01d-410bc35c9f1d)


### Frontend


### Backend
